classes/class.assViPLabGUI.php

```diff
@@ -719,12 +682,8 @@ public function getSolutionOutput($active_id, $pass = NULL, $graphicalOutput = F
 			
 			$soltpl->setCurrentBlock('complete');
 			$soltpl->setVariable('VIP_APP_ID',$this->getViPLabQuestion()->getId());
-			$soltpl->setVariable('VIP_APPLET_URL',$this->getPlugin()->getDirectory().'/templates/applet/TeacherApplet.jar');
-			$soltpl->setVariable('VIP_ECS_URL', ilECSSetting::getInstanceByServerId($settings->getECSServer())->getServerURI());
-			$soltpl->setVariable('VIP_WIDTH',$settings->getWidth());
-			$soltpl->setVariable('VIP_HEIGHT',$settings->getHeight());
-			$soltpl->setVariable('VIP_APPLET',$this->getPlugin()->getDirectory().'/templates/applet/TeacherApplet.jar');
-			$soltpl->setVariable('VIP_ECS_URL', ilECSSetting::getInstanceByServerId($settings->getECSServer())->getServerURI());
+			$soltpl->setVariable('VIP_ECS_URL', $settings->getECSServer()->getServerURI());
+			$soltpl->setVariable('VIP_ECS_URL', $settings->getECSServer()->getServerURI());
```

Comment:
This looks kind of wrong. Do we need to set the same variable twice?

Reply:
https://github.com/TIK-NFL/ViPLab/pull/16/files/be280af9ff5e3195128a66c43f151e33c5f5eb08#diff-737d7856fd5262b4fe171d495a56987eL723
https://github.com/TIK-NFL/ViPLab/pull/16/files/be280af9ff5e3195128a66c43f151e33c5f5eb08#diff-737d7856fd5262b4fe171d495a56987eL727